src/hooks/use-sync-states-progress-info.ts

```diff
@@ -24,6 +29,18 @@ export type SyncBackupResponse = {
 	percent: number;
 };
 
+export type ImportResponse = {
+	status:
+		| 'finished'
+		| 'failed'
+		| 'initial_backup_started'
+		| 'archive_import_started'
```

Comment:
I see this status is new.

Reply:
Yes, I added `archive_import_finished` to [handle it here](https://github.com/Automattic/studio/blob/7540a42357ae5e1c3fb261ef507bc45628762ec7/src/hooks/use-sync-states-progress-info.ts#L207), but as @wojtekn  [asks here](https://github.com/Automattic/studio/pull/1389#issuecomment-2880570952), we can keep the progress under 100 % during that status and the `finished` one. What do you think?